README.md

```diff
@@ -11,19 +11,23 @@ extensible to support any kind of data stores.
 The formal definition of the Criteria4s' type-classes (expressions) is as follows:
 
 ```text
-Criteria    :=  ConjOp Criteria Criteria | PredOp Ref Ref | Value<Boolean>
-Ref         :=  Value<T> | Col
-ConjOp      :=  AND | OR
-PredOp      :=  EQ | NEQ | GT | LT | GEQ | LEQ | IN | LIKE | IS_NULL | IS_NOT_NULL ...
+Criteria     :=  ConjOp Criteria Criteria | RefExpr | Value<Boolean>
+ConjOp       :=  AND | OR
+RefExpr      :=  UnPredOp Ref | BinPredOp Ref Ref | RefColOp Ref Collection<V> | RefRangeOp Ref Range<V> | RegexOp Ref Value<String>
+Ref          :=  Value<T> | Col
+UnPredOp     :=  IS_NULL | IS_NOT_NULL
+BinPredOp    :=  EQ | NEQ | GT | LT | GEQ | LEQ
+RefColOp     :=  IN | NOTIN
+RefRangeOp   :=  BETWEEN | NOTBETWEEN
+RegexOp      :=  LIKE
 ```
```

Comment:
Well, I tried to fix this Backus–Naur form in order to describe actual form of criteria
But now it looks a little bit complicated and actually I think it should be (just look at the formal expressions of postgresql expressions😁)

In my opinion, we could exclude this theoretical part from README and add more practical examples below🙂

What do you think?

Reply:
okay, I get it, then I'll make it as you say :)